app/lib/features/news/providers/news_post_editor_providers.dart

```diff
@@ -104,6 +106,18 @@ class NewsStateNotifier extends StateNotifier<NewsPostState> {
     state = state.copyWith(currentNewsSlide: selectedNewsSlide);
   }
 
+  Future<void> selectChatToShare(BuildContext context) async {
+    final selectedChatId = await selectChatDrawer(context: context);
+    RefDetails? refDetails;
+    if (selectedChatId != null) {
+      final selectedChat = await ref.read(chatProvider(selectedChatId).future);
+      refDetails = await selectedChat!.refDetails();
```

Comment:
possible null check before (!) to avoid it:
```suggestion
      if(selectedChat == null) return;
      refDetails = await selectedChat.refDetails();
```